blackboxopt/visualizations/visualizer.py

```diff
@@ -195,6 +196,117 @@ def multi_objective_visualization(
     return fig
 
 
+def parallel_coordinate_plot_parameters(
+    evaluations: List[Evaluation],
+    columns: Optional[List[str]] = None,
+    color_by: Optional[str] = None,
+):
+    """Create an interactive parallel coordinate plot.
+
+    Useful to investigate relationships in a higher dimensional search space and the
+        optimization's objective(s).
+
+    Args:
+        evaluations: Evaluations to plot.
+        columns: Names of columns to show. Can contain parameter names, objective names
+            and settings keys. If `None`, all parameters, objectives and settings are
+            displayed.
+        color_by: Parameter name, objective name, settings key. The corresponding column
```

Comment:
Reading this, it is not clear to me if the value for `color_by` needs to already be in `columns` or whether it will be plotted twice then. I can figure this out by looking at the code, but from the docstring, this remains unclear.

Reply:
Currently it behaves in the most flexible way, meaning `columns=["a","b","c"], color_by="d"`will show only three columns (a, b, c), but they will be colored by the value of d. 

I don't know if this is of any use, and also in this case the visualization is a bit misleading with having the color-bar according to d right next to the most right column c, that's why I didn't want to actually "promote" this way of usage by mentioning it explicitly that there is no such restriction.

What do you think about that? Should we enforce that `color_by` is included in the displayed `columns`? Should we make it more explicit in the docstring, that there is no such restriction?